packages/history/src/models/migrations/20200320081228-insert-WorkflowTransHist-indexes.js

```diff
@@ -5,14 +5,20 @@ const FINISHEDON_INDEX = 'WorkflowTransitionHistory_FinishedOn';
 
 module.exports = {
   up: (queryInterface) => {
-    queryInterface.addIndex(TABLE, {
-      fields: ['businessObjId', 'businessObjType', 'to'],
-      name: SEARCH_INDEX
-    });
-    queryInterface.addIndex(TABLE, {
-      fields: ['finishedOn'],
-      name: FINISHEDON_INDEX
-    });
+    queryInterface.addIndex(
+      TABLE,
+      ['businessObjId', 'businessObjType', 'to'],
+      {
+        name: SEARCH_INDEX
+      }
+    );
+    queryInterface.addIndex(
+      TABLE,
+      ['finishedOn'],
```

Comment:
Please revert this change, because according to https://sequelize.org/master/class/lib/dialects/abstract/query-interface.js~QueryInterface.html#instance-method-addIndex previous definition is recommended:
![Screenshot 2021-09-22 at 13 25 20](https://user-images.githubusercontent.com/31243790/134327484-b91e3431-86df-410c-b086-e0bf56f471ba.png)